### ISSUE TITLE
Translate Tab Organizer to other languages.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#139]
Originally reported on Google Code with ID 140

```
I mostly know English, so I'll need help with this. I can do the programming bits, but
I'll need other people to translate the text into other languages.

If you're interested, please post a comment in here, and I'll get you a list of the
things that need translating.

Also, would it be a good idea to use translate.google.com? I don't think their translations
are always correct, though.
```

Reported by `pcxunlimited` on 2011-02-28 18:17:14

**Comment by Pauan** [comment]
```
http://tab-organizer.googlecode.com/hg/_locales/en/messages.json

The above link should contain the most up-to-date English localization for Tab Organizer.

All you need to do is change the stuff after "message" into a different language. So
for instance:

"global_extension_name": {
  "message": "Tab Organizer",
  "description": "The name of the extension."
},

Would become the following, in Finnish:

"global_extension_name": {
  "message": "Tab Järjestäjä"
},

You don't need to translate the "description" part, but I guess you can if you really
want to. It will probably help a lot if you're familiar with Tab Organizer. You shouldn't
need to understand the code, but you should know which text goes where.

So for instance, you should know that "Menu" refers to the button in the upper-left
corner, that "You selected " is displayed in the undo bar, etc.
```

Reported by `pcxunlimited` on 2011-02-28 21:40:16

**Comment by Pauan** [comment]
```
Also, you don't need to translate them all at once. If doing them one at a time is better
for you, then go for it. It should work even if it's only partially translated.
```

Reported by `pcxunlimited` on 2011-02-28 21:49:25

**Comment by Pauan** [comment]
```
Oh yeah, and some of the messages say to "underline a letter", which just means to surround
the letter with <u></u>, like this: <u>M</u>. If there isn't a suitable letter, then
don't underline it.

This is used in the menus to tell the user what key to press, like "T" for "New Tab",
"a" for "Select all" etc.
```

Reported by `pcxunlimited` on 2011-04-15 07:02:54

**Comment by Pauan** [comment]
```
I think people on the internet should just adapt English. Even though I am from Germany
I prefer my apps and extensions in English. It is just the language of the internet.
At least that is how I see it. :)
```

Reported by `didnotlearn@hotmail.de` on 2012-04-30 00:14:13

**Comment by Pauan** [comment]
```
That's all well and good, but not everyone has that luxury or desire. Of course, I can't
really do this myself, so it's entirely dependent on other people who wish to volunteer.
It's up to them whether it happens or not.
```

Reported by `pcxunlimited` on 2012-04-30 03:12:30

**Comment by Pauan** [comment]
```
Through Google I found out that I can edit .json-files with any text editor. So I guess
people should send the translated .json file to you as an e-mail attachment? 
```

Reported by `didnotlearn@hotmail.de` on 2012-04-30 11:06:56

**Comment by Pauan** [comment]
```
Sure, or attach it here, or upload it to another hosting site, I don't really care as
long as it works.
```

Reported by `pcxunlimited` on 2012-05-01 01:41:39

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:23
- **Labels added**: Type-External
- **Labels removed**: Type-Task

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:08:43
- **Status changed**: `Available`
- **Labels added**: Type-HelpNeeded
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
```
Hey, here's a Portuguese (PT-pt, not PT-br) translation. I need to point a problem out
though: the global_plural thing should be different for tabs and windows in Portuguese.
The plural of "separador" ("tab") is "separadorES", while the plural of "janela" ("window")
is "janelaS". I went with "es" because there's usually more than one tab and multiple
windows are rarer, but yeah.
There were several small problems like this and I worked around as many as possible,
but fully translating this extension correctly to Portuguese is sort of impossible
right now due to the way it's coded. It should be understandable though.
```

Reported by `mail.do.Guilherme` on 2012-09-08 13:58:17

<hr>
- _Attachment: [messages  (PT-pt version).json](https://storage.googleapis.com/google-code-attachments/tab-organizer/issue-140/comment-11/messages  %28PT-pt version%29.json)_

**Comment by Pauan** [comment]
```
Thank you very much! I'm currently in the process of rewriting Tab Organizer, so a lot
of things will change, but I'm sure I can reuse a lot of the work you've done. I'll
definitely keep in mind to split the plural into separate pieces.
```

Reported by `pcxunlimited` on 2012-09-09 02:06:49
